tests/main/security-device-cgroups-devmode/task.yaml

```diff
@@ -13,6 +13,12 @@ prepare: |
         touch /dev/fb0.spread
     fi
 
+    # Create a test char device so we can verify the test snap cannot read.
+    if [ ! -e /dev/test1 ]; then
+        mknod /dev/test1 c 29 0
```

Comment:
Just FYI, 29:0 char is framebuffer, the same as /dev/fb0

Reply:
yes, I just wanted to make sure the char device is supported by the kernel, I'll try with a different one.